tests/test_patchanalysis.py

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
This reminds me that patchanalysis is not being used by anyone AFAIK, we should maybe remove it at this point. It's likely very outdated.